CaveRunner/Assets/Skillz/Build/IncludeInXcode/Skillz+Unity.mm

```diff
@@ -45,6 +45,15 @@ static void PauseApp()
     UnityPause(true);
 }
 
+static void PauseAppWithDelay()
+{
+    double delayInSeconds = 1.0;
+    dispatch_time_t popTime = dispatch_time(DISPATCH_TIME_NOW, (int64_t)(delayInSeconds * NSEC_PER_SEC));
+    dispatch_after(popTime, dispatch_get_main_queue(), ^(void){
+        PauseApp();
```

Comment:
If we're only using PauseAppWithDelay now, could we potentially move the PauseApp statements within this method to get rid of the PauseApp method entirely?

Reply:
We're not only using this method now though, it's also being called elsewhere in this file (e.g. `_launchSkillz()`

Reply:
Ah okay, was just curious.